cmd/bee/cmd/start.go

```diff
@@ -420,36 +412,38 @@ func (c *command) configureSigner(cmd *cobra.Command, logger logging.Logger) (co
 	}, nil
 }
 
-func parseNetworks(main bool, networkID uint64) (uint64, error) {
-	if main && networkID != 1 {
-		return 0, errors.New("provided network ID does not match mainnet")
-	}
-
-	return networkID, nil
+type mainnetCfg struct {
```

Comment:
in addition to a bee network default config struct, we also should have that for blockchain (which is where the blockTime should be on). also we want an expected chain id in here.